### PR TITLE
test: add tests for `KImg`

### LINKED ISSUE #512: `KImg`: Introduce testing
## Product
KDS

## Desired behavior
The recently introduced `KImg` component must have some associated tests to ensure that future changes to the library do not break the existing features. 

## Current behavior
There are no tests associated with the `KImg` component. 

# Add labels
`testing`

### PR DESCRIPTION
**Pull request by EshaanAgg**
## Description
Adds tests for the `Kimg` component

#### Issue addressed
Fixes #512 

### Before/after screenshots
NA

## Changelog

- [#518]
  - **Description:** Add testing for KImg component
  - **Products impact:** none
  - **Addresses:** https://github.com/learningequality/kolibri-design-system/issues/512
  - **Components:** -
  - **Breaking:** no
  - **Impacts a11y:** no  - 
  - **Guidance:** -

[#518]: [https://github.com/learningequality/kolibri-design-system/pull/518]

## Steps to test

1. Run the test suite

### At a high level, how did you implement this?
Added the relevant tests

### Does this introduce any tech-debt items?
No

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [x] Critical and brittle code paths are covered by unit tests
- [x] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

**Comment by MisRob** [comment]
Thanks @EshaanAgg, I will read it in more detail in the next few days, but the scenarios you decided to test and the overall strategy makes sense to me already.

File: lib/KImg.spec.js
Comment (MisRob) [inline lib/KImg.spec.js:1]: Could you create a new folder for `KImg` with `__tests__` subfolder and its test suite into there? It's a convention we've been trying to follow. Please also move `KImg.vue` into the folder and rename it to `index.vue`.

`lib/KImg.vue` --> `lib/KImg/index.vue`
`lib/Kimg.spec.js` --> `lib/KImg/__tests__/KImg.spec.js`

 You can see `KTextbox` folder for example.

**Review by MisRob — CHANGES_REQUESTED** [review]
Thanks @EshaanAgg, it's a really good test suite, easy to navigate and understand. Your selection of tests scenarios makes sense to me.

I'd like to request only one change in regards to organization of the test file before merge.

File: lib/KImg.spec.js
Comment (EshaanAgg) [reply lib/KImg.spec.js:1]: Made these changes!

Comment (MisRob) [reply lib/KImg.spec.js:1]: Thanks, looking good. Would you mind also resolving the conflicts?

Comment (EshaanAgg) [reply lib/KImg.spec.js:1]: Done!

**Review by MisRob — APPROVED** [review]
Good work, thank you!

**Pull request merged**